src/Microsoft.Health.Fhir.TemplateManagement.FunctionalTests/OCIArtifactFunctionalTests.cs

```diff
@@ -42,67 +43,68 @@ public class OCIArtifactFunctionalTests
         private readonly string _testMultiLayersWithInValidSequenceNumberImageReference;
         private readonly string _testInvalidCompressedImageReference;
         private bool _isOrasValid = true;
-        private string _orasErrorMessage;
-        private readonly string _orasFileName;
+        private string _orasErrorMessage = "Oras tool invalid.";
 
         public OCIArtifactFunctionalTests()
         {
-            _orasFileName = "oras.exe";
             _containerRegistryServer = "localhost:5000";
             _testOneLayerWithValidSequenceNumberImageReference = _containerRegistryServer + "/templatetest:onelayer_valid_sequence";
             _testOneLayerWithoutSequenceNumberImageReference = _containerRegistryServer + "/templatetest:onelayer_without_sequence";
             _testOneLayerWithInValidSequenceNumberImageReference = _containerRegistryServer + "/templatetest:onelayer_invalid_sequence";
             _testMultiLayersWithValidSequenceNumberImageReference = _containerRegistryServer + "/templatetest:multilayers_valid_sequence";
             _testMultiLayersWithInValidSequenceNumberImageReference = _containerRegistryServer + "/templatetest:multilayers_invalid_sequence";
             _testInvalidCompressedImageReference = _containerRegistryServer + "/templatetest:invalid_image";
-            PushOneLayerWithValidSequenceNumber();
-            PushOneLayerWithoutSequenceNumber();
-            PushOneLayerWithInvalidSequenceNumber();
-            PushMultiLayersWithValidSequenceNumber();
-            PushMultiLayersWithInValidSequenceNumber();
-            PushInvalidCompressedImage();
+            Task.Run(PushOneLayerWithValidSequenceNumberAsync).Wait();
```

Comment:
better to use ```IAsyncLifetime``` to execute initialization steps.

Reply:
Fixed it.